src/packtPublishingFreeEbook.py

```diff
@@ -151,13 +155,11 @@ def get_ebook_infodata(self, r):
         :return: the data ready to be written to the log file
         """
         logger.info("Retrieving complete information for '{}'".format(self.book_title))
-        r = self.session.get(self.account_data.freelearning_url,
-                             headers=self.account_data.req_headers, timeout=10)
+        r = self.session.get(PACKTPUB_FREE_LEARNING_URL, headers=PACKTPUB_REQUEST_HEADERS, timeout=10)
         result_html = BeautifulSoup(r.text, 'html.parser')
         last_grabbed_book = result_html.find('div', {'class': 'dotd-main-book-image'})
         book_url = last_grabbed_book.find('a').attrs['href']
-        book_page = self.session.get(self.account_data.packtpub_url + book_url,
-                                     headers=self.account_data.req_headers, timeout=10).text
+        book_page = self.session.get(PACKTPUB_URL + book_url, headers=PACKTPUB_REQUEST_HEADERS, timeout=10).text
```

Comment:
Poddaje pod rozwagę użycie ```urljoin``.

Reply:
That's obviously a better option - I'll do as you say.

Reply:
`urllib` is specific for Python 3, but I'll try to figure something out. I'm using Python 3 as well, but this script should be compatible with Python 2 also (but I don't know if it's true).

Reply:
These is no new python library. There is only other structure and import path of std. See https://github.com/search?utf8=%E2%9C%93&q=urljoin+ImportError&type=Code .